docs/02-guides/server-side-events/create-an-integration.mdx

```diff
@@ -7,7 +7,135 @@ sidebar_position: 1
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+This guide will help you to create an integration that will dispatch events to external services.
+
+We will create a new integration to dispatch events to [Wehook.cool](https://webhook.cool/).
+
+:::info
+
+Front-Commerce already provide a Webhook integration. This guide will show you how to create an integration from scratch. And will explain you the basics of integrations so you can write your own
+
+:::
+
+## Create an integration file
+
+First you need to create an integration file let's say we will call our integration Coolwebhook
+
+Let's create a new file in `app/server-evets/integrations/CoolWebhookIntegration.ts`
+
+Integrations are classes that extends `Integration interface`, let's make a simple copy paste of this code:
+
+```ts title="app/server-evets/integrations/CoolWebhookIntegration.ts"
+import type { Integration } from "@front-commerce/core/server-events/integrations";
+
+export default class CoolWehbookIntegration implements Integration {
+  compatibleEvents: string[] | "*" = "*";
+
+  webhookId: string;
+
+  constructor(webhookId: string) {
+    this.webhookId = webhookId;
+  }
+
+  handle(messageId: string, payload: any) {
+    console.log(messageId);
+    console.log(payload)
+  }
+}
+```
+
+In this class, we have four entities :
+- `compatibleEvents`: this indicates which events the integration will handle, it can be `*` to handle all events or an array of string that contains compatible events.
+- `webhookId`: this is the id of WebhookCool.
```

Comment:
**nitpick:** I haven't understood what it meant at first read

**suggestion:** provide an example

```suggestion
- `webhookId`: this is the id of the unique WebhookCool url to use (e.g: `hissing-dentist-09` for https://webhook.cool/at/hissing-dentist-09)
```